packages/components/src/font-size-picker/test/utils.ts

```diff
@@ -80,74 +76,3 @@ describe( 'splitValueAndUnitFromSize', () => {
 		}
 	);
 } );
-
-describe( 'getToggleGroupOptions', () => {
```

Comment:
I don't think these tests add much value beyond what's already tested in `test/index.tsx`.

Reply:
I had a look and couldn't really see a test checking for t-shirt sizes. Could you add a test for that in `test/index.tsx` ?

Reply:
👍 added a test that checks all the t-shirt labels are used even when there's no font size names.